alacritty_macos.yml

```diff
@@ -43,12 +43,18 @@ font:
 # Should display the render timer
 render_timer: false
 
+# Should invert the cursor color. If false, display the cursor in the
+# cursorforeground and cursorbackground colors
+invert_cursor: true
+
 # Colors (Tomorrow Night Bright)
 colors:
   # Default colors
   primary:
     background: '0x000000'
     foreground: '0xeaeaea'
+    cursorbackground: '0xff0000'
```

Comment:
These should be in a separate field `cursor` and have `foreground` and `background`.